.travis.yml

```diff
@@ -15,14 +16,11 @@ matrix:
   include:
     - php: 5.3
       env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
-    - php: 5.6
-      env: SYMFONY_VERSION=2.3.* SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 5.6
-      env: SYMFONY_VERSION=2.8.*
-    - php: 5.6
-      env: SYMFONY_VERSION=3.0.*
-  allow_failures:
-    - php: 5.6
+    - php: 7.0
+      env: SYMFONY_VERSION=2.3.*
+    - php: 7.0
+      env: SYMFONY_VERSION=2.7.*
+    - php: 7.0
       env: SYMFONY_VERSION=3.0.*
```

Comment:
PHP 7 builds should be removed from allowed failures imo.

Also, this can be simplified by one line: `- php: 7.0` meaning everything with PHP 7 will be allowed to fail.